pgnReader.py

```diff
@@ -0,0 +1,38 @@
+'''

```

Comment:
I recommend formatting the code (e.g. via black https://github.com/psf/black) so that the format is consistent

what about adding a Bash script that automatically formats all Python code and running it before commit?